save-backend/src/test/resources/minio/cosv/5

```diff
@@ -21,6 +21,9 @@
     }
   ],
   "summary": "Incorrect Authorization",
+  "cwe_ids": ["1", "42", "308"],
```

Comment:
I guess `cwe_ids` is `CWE-<N>`, and `cwe_names` is short description.

like here https://cwe.mitre.org/top25/archive/2023/2023_stubborn_weaknesses.html

Reply:
I though about it, but I was little bit confused by `CWE Name` and not `CWE Description`

changed